backend/api/users.py

```diff
@@ -30,7 +30,6 @@ def logout():
 
 
 @users.route('/api/users', methods=['POST'])
-@roles_required('admin')
```

Comment:
@roles_required('admin') currently does nothing in blueprints anyways so the change was a bit pointless. it was now changed to @admin_permission.require(), which should equally not be removed from master until decided otherwise by everyone together